### DISCUSSION TITLE
Need extra channels. Use Phase C for standard measuement?

### DISCUSSION THREAD

**Discussion by copekyle** [emporia-vue-local/esphome#72] [Q&A] [answered]
My main meter box has 40 breakers. I've combine the circuits that make sense and I'm still needing 34 circuits (ct's). I have 2 vue's to work with.

I'm wondering if I can use the 200 amp Phase C port of each device for CT measurements? I would assume that I'd need to hook up the blue vue power wire to the phase that I'm measuring? or can I just reference what ever phase voltage the ct is on in the software? I don't want to do this and destroy my vue's. I can figure out the software with guess and check but I only get one shot with the hardware.

Assuming it's just code, would it look like this?
```
 - { phase_id: phase_a, input:  "C", power: { name:  "Living Room Power", id:  cir1, filters: [ *moving_avg, *pos ] } }
```

**Comment by flaviut** [comment, marked as answer]
> I'm wondering if I can use the 200 amp Phase C port of each device for CT measurements?

Yup!

> I would assume that I'd need to hook up the blue vue power wire to the phase that I'm measuring? or can I just reference what ever phase voltage the ct is on in the software?

No need for extra wiring, you can just reference the correct phase from the software. And your example config is right.

> I can figure out the software with guess and check but I only get one shot with the hardware.

Hooking it up wrong won't break anything.

**Comment by copekyle** [comment]
Follow up to that. Can I use the 50 amp ct in Phase C as long as I have a 3.5mm to 2.5mm adapter?

It does look like emporia is out of individual 50amp sensors, are there any alternatives that can be used?

**Reply by flaviut** [reply]
> Can I use the 50 amp ct in Phase C as long as I have a 3.5mm to 2.5mm adapter?

You'll probably need to calibrate using a custom multiplier in the filters field, but it will work fine.

> are there any alternatives that can be used?

See https://github.com/emporia-vue-local/esphome/discussions/59

**Comment by ziptbm** [comment]
@copekyle did you end up configuring Port C successfully?  If so, could you please share your config?  

I'm considering this as an option to measure a 100A dual-pole circuit that I'm not getting accurate readings from with the normal side ports.  So I'm exploring options to measure it more accurately and have an extra 200A CT.

**Reply by copekyle** [reply]
I did not. I ended up having to purchase an individual device with 2 ct's to measure an unbalanced dual-pole circuit. I ended up using a Shelly EM for just that circuit.

https://www.shelly.cloud/en-us/products/product-overview/em-120a/em-2x120a
